src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -374,6 +369,26 @@ def load_arguments(self, _):
             c.argument('enable', options_list=['--enable-cd', '-e'], help='enable/disable continuous deployment',
                        arg_type=get_three_state_flag(return_label=True))
 
+    with self.argument_context('webapp config container') as c:
+        c.argument('docker_registry_server_url', options_list=['--docker-registry-server-url', '-r'],
+                   help='the container registry server url')
+        c.argument('docker_custom_image_name', options_list=['--docker-custom-image-name', '-c', '-i'],
+                   help='the container custom image name and optionally the tag name (e.g., <registry-name>/<image-name>:<tag>)')
+        c.argument('docker_registry_server_user', options_list=['--docker-registry-server-user', '-u'],
+                   help='the container registry server username')
+        c.argument('docker_registry_server_password', options_list=['--docker-registry-server-password', '-p'],
+                   help='the container registry server password')
+
+    with self.argument_context('functionapp config container') as c:
+        c.argument('registry_server', options_list=['--registry-server', '-r', c.deprecate(target='--docker-registry-server-url', redirect='--registry-server')],
+                   help='the container registry server url')
+        c.argument('image', options_list=['--image', '-c', '-i', c.deprecate(target='--docker-custom-image-name', redirect='--image')],
+                   help='the container custom image name and optionally the tag name (e.g., <registry-name>/<image-name>:<tag>)')
+        c.argument('registry_username', options_list=['--registry-username', '-u', c.deprecate(target='--docker-registry-server-user', redirect='--registry-username')],
+                   help='the container registry server username')
+        c.argument('registry_password', options_list=['--registry-password', '-p', c.deprecate(target='--docker-registry-server-password', redirect='--registry-password')],
+                   help='the container registry server password')
```

Comment:
Why do you want to rename these option names?

Reply:
We want to rename these option names for two reasons:
1. They currently give the impression that only docker is supported, but ACR is also supported
2. We want to be consistent with [container apps commands](https://learn.microsoft.com/en-us/cli/azure/containerapp?view=azure-cli-latest#az-containerapp-create) which have the option names `registry-password`, `registry-username` and `registry-server`. 
At the same time, we want to make sure that there is backward compatibility for the customers who have created scripts that rely on the old naming conventions, which is why we just marked the old option names as deprecated.